tests/e2e.test.ts

```diff
@@ -14,15 +14,16 @@ import {
   PredicateType,
   CredentialState,
   ProofState,
-} from '../src'
+} from '../packages/core/src'
```

Comment:
Could change to from `from '@aries-framework/core'`?

Reply:
Yes!